src/main/java/uk/gov/justice/digital/hmpps/cmd/api/service/NotificationService.kt

```diff
@@ -60,8 +61,25 @@ class NotificationService(
 
         allPrisons.forEach { prison ->
             saveNotifications(
-                    csrClient.getShiftTaskNotifications(prison.csrPlanUnit, prison.region)
-                            .distinct())
```

Comment:
The distinct we did here was too simple, we actually need to apply some more complicated logic.
I've also moved a method that shouldn't have been in the web client.